SendGrid/Example/Example.cs

```diff
@@ -35,13 +35,13 @@ private static void Main()
             String subject = "I'm replacing the subject tag";
             Email to = new Email("elmer@sendgrid.com");
             Content content = new Content("text/html", "I'm replacing the <strong>body tag</strong>");
-            Mail mail = new Mail(from, subject, to, content);
+            SendGrid.Models.Mail mail = new SendGrid.Models.Mail(from, subject, to, content);
 
             mail.TemplateId = "13b8f94f-bcae-4ec6-b752-70d6cb59f932";
             mail.Personalization[0].AddSubstitution("-name-", "Example User");
             mail.Personalization[0].AddSubstitution("-city-", "Denver");
 
-            dynamic response = await sg.client.mail.send.post(requestBody: mail.Get());
+            dynamic response = await sg.client.mail.send.post(requestBody: mail.Serialize());

```

Comment:
@thinkingserious I'd say I'm not able to get this change until you accept your pull request and you merge it to the branch